opensrp-chw-core/src/test/java/org/smartregister/chw/core/utils/BAJsonFormUtilsTest.java

```diff
@@ -97,6 +107,45 @@ public void getAutoJsonEditMemberFormStringReturnsCorrectJSONObject() throws JSO
         Assert.assertEquals(formTitle, stepOne.getString(TITLE));
     }
 
+    @Test
+    public void getStartFormActivityReturnsCorrectIntent() {
+        Context context = RuntimeEnvironment.application;
+        Intent testIntent = FormUtils.getStartFormActivity(new JSONObject(), "test form", context);
+        Assert.assertNotNull(testIntent);
+        Form form = (Form) Objects.requireNonNull(testIntent.getExtras()).get(JsonFormConstants.JSON_FORM_KEY.FORM);
+        assertEquals("test form", form.getName());
+    }
+
+    @Test
+    public void testComputeDOBUnknown() {
```

Comment:
Kindly follow the test naming convention `test[MethodName][ExpectedResult][Parameters]`